src/main/kotlin/day19/data_source/NotificationDataSource.kt

```
@@ -0,0 +1,4 @@
+package org.example.day19.data_source
+
+interface NotificationDataSource {
+}
```

detekt (1.23.7)

[warning] The class or object NotificationDataSource is empty.

(detekt.empty-blocks.EmptyClassBlock)